dev/DynamicDependency/DataStore.cpp

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "DataStore.h"
+
+#include "DynamicDependencyDataStore_h.h"
+#include "utf8.h"
+#include "winrt_msixdynamicdepednency.h"
+
+#include <wil/winrt.h>
+
+#include <shlobj.h>
+
+MddCore::PackageDependency MddCore::DataStore::Load(PCWSTR packageDependencyId)
+{
+    std::filesystem::path relativeFilename{ L"DynamicDependency" };
+    relativeFilename /= std::wstring(packageDependencyId) + DataStore::fileExtension;
```

Comment:
this does an in-place append, right?  (+= does concatenate, creates a new object?)

Reply:
Correct. `std::filesystem::path /= x;` is akin to `std::string += "\" + x;` with all the additional smarts/complexities of dealing in paths rather than merely strings. Very handy